spec/models/green_flag/feature_spec.rb

```diff
@@ -107,7 +107,7 @@
      end
    end
 
-   describe 'latest_version' do
+   describe '#latest_version' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.